core/src/main/resources/META-INF/services/com.google.cloud.tools.jib.api.DockerClient

```diff
@@ -0,0 +1 @@
+org.testcontainers.containers.JibDockerClient
```

Comment:
I don't think we should implement the SPI. It is too implicit, **especially** given that `supported` returns `true` unconditionally.

I think we should just provide a singleton instance anyone can use with Jib explicitly.

Reply:
The reason to use the SPI was to delegate everything under the same behavior. By default, the Jib implementation will use docker cli. I was looking for a way to hide it.